src/analyses/mutexAnalysis.ml

```diff
@@ -47,6 +49,15 @@ struct
   module G = P.G
   module V = VarinfoV
 
+  module GM = Hashtbl.Make (ValueDomain.Addr)
+  module VarSet = SetDomain.Make (Basetype.Variables)
+  let gm_rw = GM.create 10 (* TODO: marshal? move into global invariant? *)
+  let gm_w = GM.create 10 (* TODO: marshal? move into global invariant? *)
```

Comment:
Currently these hashtables are inside the `Spec`, but maybe they should be moved into the global invariant instead? Right now, this information isn't correctly handled during incremental analysis.